escalation/ticketEscalation.sw.yaml

```diff
@@ -87,13 +87,14 @@ states:
         functionRef:
           refName: createNotification
           arguments:
-            title: '"ATTN: Escalation for ticket - " + .createResponse.ticketId'
-            message: '"Please manage escalation ticket " + .createResponse.ticketId '
-            origin: "Escalation Workflow"
-            topic: "Escalation Workflow"
-            actions:
-              - title: '"Escalation ticket " + .createResponse.ticketId'
-                url: .createResponse.browseUrl
+            recipients:
+              type: "broadcast"
```

Comment:
there's a story opened for this. https://issues.redhat.com/browse/FLPATH-1233. do u want to fix it here?